README.md

```diff
@@ -1,74 +1,89 @@
-    | code            | For actions related to building code     |
-    | certs           | For actions related to certificates      |
-    | db              | For actions related to databases         |
-    | git             | For actions related to Git               |
-    | iis             | For actions related to IIS               |
-    | microsoft-teams | For actions related to Microsoft Teams   |
-    | svc             | For actions related to Windows Services  |
-    | jira            | For actions related to Jira              |
-    | meta            | For actions related to running workflows |
-    | pagerduty       | For actions related to PagerDuty         |
-    | test            | For actions related to testing           |
-    | tf              | For actions related to Terraform         |
-  - [ ] Add any additional topics for an action if they apply    
-    
+# IIS Service Action
+
+A GitHub action that can start, stop, or restart an On-Prem IIS servers.
+
+## Index <!-- omit in toc -->
+
+- [Inputs](#inputs)
+- [Prerequisites](#prerequisites)
+- [Example](#example)
+- [Code of Conduct](#code-of-conduct)
+- [License](#license)
 
 ## Inputs
-| Parameter | Is Required | Description           |
-| --------- | ----------- | --------------------- |
-| `input-1` | true        | Description goes here |
-| `input-2` | false       | Description goes here |
 
-## Outputs
-| Output     | Description           |
-| ---------- | --------------------- |
-| `output-1` | Description goes here |
+| Parameter                  | Is Required | Description                                              |
+| -------------------------- | ----------- | -------------------------------------------------------- |
+| `server`                   | true        | The name of the target server                            |
+| `service-account-id`       | true        | The service account name                                 |
+| `service-account-password` | true        | The service account password                             |
+| `app-pool-name`            | true        | IIS app pool name                                        |
+| `action`                   | true        | Specify start, stop, or restart as the action to perform |
+| `server-public-key`        | true        | Path to remote server public ssl key                     |
+
+## Prerequisites
+
+Inbound secure WinRm network traffic (TCP port 5986) must be allowed from the GitHub Actions Runners virtual network so that IIS service commands from the runners  can be received.
+
+Prep the remote IIS server to accept WinRM IIS management calls.  In general the IIS server needs to have a Web Services for Management, [WSMan], listener that looks for incoming Windows Remote Management, [WinRM], calls. Firewall exceptions need to be added for the secure WinRM TCP ports, and non-secure firewall rules should be disabled. Here is an example script that would be run on the IIS server:
```

Comment:
Thank you for clarifying, I thought this was two separate things before....I mean other people thought that.  👀

Reply:
No problem.  It was a complicated action to produce and verify.  It warrants more explanation so that it doesn't seem too obscure to troubleshoot later on.